lefthook.yml

```diff
@@ -8,3 +8,7 @@ pre-commit:
       glob: "*.py"
       run: uv run ruff format {staged_files}
       stage_fixed: true
+    prettier:
+      glob: "*.{yml,yaml,md}"
```

Comment:
Is there a clean way to share this configuration (yml, yaml, md) between the linter CI job and lefthook?

It's OK if there is not, but please check